### PR TITLE
Bugfix for search scoring config

### PR DESCRIPTION
**Pull request by alat**
Config attributes were taken from wrong place

### CODE REVIEW

**Review by cewald — APPROVED** [review]

File: src/elasticsearch/score.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ export default function getFunctionScores (config) {
   }
   let filter = []
   let esScoringAttributes = config.elasticsearch.searchScoring.attributes
+  let esScoringConfig = config.elasticsearch.searchScoring
```
Comment (cewald) [inline src/elasticsearch/score.ts:7]: Just to save like 5 characters ;)
```suggestion
  let esScoringConfig = config.elasticsearch.searchScoring
  let esScoringAttributes = esScoringConfig.searchScoring.attributes
```

**Comment by cewald** [comment]
@pkarw @gibkigonzo I can approve, this a bug and its solution :) I could merge it if it's okay?

File: src/elasticsearch/score.ts (hunk 1 above)
Comment (gibkigonzo) [reply src/elasticsearch/score.ts:7]: I think it should be like this 🤔 
```
  let esScoringConfig = config.elasticsearch.searchScoring
  let esScoringAttributes = esScoringConfig.attributes
```

**Review by gibkigonzo — APPROVED** [review]

File: src/elasticsearch/score.ts (hunk 1 above)
Comment (cewald) [reply src/elasticsearch/score.ts:7]: @alat Can you commit the suggested changes?

**Review by Fifciu — APPROVED** [review]

**Pull request merged**